code/drasil-printers/lib/Language/Drasil/Markdown/Helpers.hs

```diff
@@ -62,9 +62,16 @@ reflinkURI ref txt = if ref == txt then ang ref
 
 -- | Helper for setting up figures
 image :: Doc -> Doc -> Doc
-image f c =  text "!" <> reflinkURI fp c $^$ bold c
+image f c =  text "!" <> reflinkURI fp c $^$ figCaption c
   where
     fp = text $ "./assets/" ++ takeFileName (show f)
+    figCaption cap
```

Comment:
The reasoning for `figCaption` should be the same here as for HTML, no? Perhaps even share the same code for the part that emits "Figure: " ?